### PR TITLE
DAT-3789 | Infobox Builder - source attribute immutable after first save

### PR DESCRIPTION
**Pull request by rybmat**
**Work in progress. Don't merge!** State it here and use labels.

## Links

* https://wikia-inc.atlassian.net/browse/DAT-3789

## Description

Prevent infobox builder from changing source attribute of row elements in existing infoboxes

## Reviewers
@RafLeszczynski @dianafa

### CODE REVIEW

File: front/main/tests/unit/models/infobox-builder-test.js [hunk 1]
```diff
@@ -63,13 +63,36 @@ test('add items by type', (assert) => {
...
-					type: 'row'
+					type: 'row',
+					sourceFreezed: false
 				},
 				message: 'add row item'
 			},
 			{
 				dataMock: {
 					data: {
+						label: messageMock
```
Comment (RafLeszczynski) [inline front/main/tests/unit/models/infobox-builder-test.js]: I wander if we need this extending test here as we already have test for extending rows in a separate test case below. What do you think?

**Comment by RafLeszczynski** [comment]
one small concern, except it LGTM and please merge newest dev, run linter and wait for build to pass before moving to QA

File: front/main/app/models/infobox-builder.js [hunk 2]
```diff
@@ -81,7 +81,8 @@ const InfoboxBuilderModel = Ember.Object.extend({
 				component: InfoboxBuilderModel.createComponentName(itemType)
 			},
 			source: `${itemType}${index}`,
-			type: itemType
+			type: itemType,
+			sourceFreezed: false
```
Comment (rogatty) [inline front/main/app/models/infobox-builder.js]: Minor - there is no such word as "freezed", it should be "frozen".
http://english.stackexchange.com/questions/23991/whats-the-proper-past-tense-of-to-freeze-freezed-or-frozen

Comment (RafLeszczynski) [inline front/main/app/models/infobox-builder.js]: your're absolutely right :-)

**Pull request merged**